rust/agama-network/src/nm/model.rs

```diff
@@ -33,18 +33,26 @@ impl NmWirelessMode {
     }
 }
 
-impl From<NmWirelessMode> for WirelessMode {
-    fn from(value: NmWirelessMode) -> Self {
+impl TryFrom<NmWirelessMode> for WirelessMode {
+    type Error = NmError;
+
+    fn try_from(value: NmWirelessMode) -> Result<Self, Self::Error> {
         match value.as_str() {
-            "infrastructure" => WirelessMode::Infra,
-            "adhoc" => WirelessMode::AdHoc,
-            "mesh" => WirelessMode::Mesh,
-            "ap" => WirelessMode::AP,
-            _ => WirelessMode::Other,
+            "infrastructure" => Ok(WirelessMode::Infra),
+            "adhoc" => Ok(WirelessMode::AdHoc),
+            "mesh" => Ok(WirelessMode::Mesh),
+            "ap" => Ok(WirelessMode::AP),
```

Comment:
I miss in this commit also removal of `Other` from `WirelessMode` enum. ( or is there reason to keep it? )

Reply:
Right, I forgot to submit such a change. Fixed now.